python/src/aiconfig/editor/client/src/components/SettingsPropertyRenderer.tsx

```diff
@@ -398,9 +408,9 @@ export default function SettingsPropertyRenderer({
               propertyDescription={propertyDescription}
             />
             <div style={{ minWidth: "350px" }}>
-              <JSONEditor
+              <JSONRenderer
```

Comment:
JSONRenderer already handles readOnly, otherwise renders JSONEditor. Essentially a no-op here when not readonly

Reply:
niceee